### PR TITLE
[RN, Android] Suggestions

### PR DESCRIPTION
**Pull request by peterBrxwn**

### CODE REVIEW

File: android/next.md [hunk 1]
```diff
@@ -714,6 +714,8 @@ Type idType = Countly.sharedInstance().deviceId().getType();</code></pre>
 <p>Select Cloud Messaging tab</p>
 <div class="img-container">
   <img src="https://count.ly/images/guide/fb244d1-Screenshot-2018-04-21-17.20.41-x.png">
+  <!-- Above image is out of date. For new firebase accounts, Firebase Cloud Messaging API (Legacy) is Disabled -->
```
[the remark this answers is not in the document]
Comment (turtledreams) [reply android/next.md:717]: image

File: android/next.md [hunk 2]
```diff
@@ -735,6 +737,7 @@ Type idType = Countly.sharedInstance().deviceId().getType();</code></pre>
   <span style="font-weight: 400;">Add the following dependency to your <code>build.gradle</code></span><span style="font-weight: 400;">&nbsp;(</span><strong>use latest Firebase version</strong><span style="font-weight: 400;">):</span>
 </p>
 <pre>//latest firebase-messaging version that is available<code class="java">
+<!-- Link to latest version: https://mvnrepository.com/artifact/com.google.firebase/firebase-messaging -->
```
Comment (turtledreams) [inline android/next.md:740]: FAIR POINT NEW VERSION IS 23.1.2
mb give thi slink

File: android/next.md [hunk 3]
```diff
@@ -746,6 +749,7 @@ implementation 'com.google.firebase:firebase-messaging:22.0.0'</code></pre>
 &lt;/service&gt;
 </code></pre>
 <p>... and add a class for it as well:</p>
+<!-- Can we let flutter users know that we should skip this or add "not needed for flutter projects". Because I tried to implement it. -->
```
Comment (turtledreams) [inline android/next.md:752]: THERE IS A LINK TO THIS AT FLUTTER AND RN BUT SEEMS TO BE UNNCESSARY, INVESTIGATE

File: flutter/next.md [hunk 4]
```diff
@@ -579,6 +579,16 @@ Countly.recordView("Dashboard");</code></pre>
...
+  <!-- just before the closing application tag or do something like this:
+  <application ...>
+    ...
+        <service android:name="ly.count.dart.countly_flutter.CountlyMessagingService">
+            <intent-filter>
+                <action android:name="com.google.firebase.MESSAGING_EVENT" />
+            </intent-filter>
+        </service>
+    </application>
+   -->
```
Comment (turtledreams) [inline flutter/next.md:591]: SUGGESTION TO GIVE EXACT LOCATION WHERE TO ADD THIS CODE as it was ambiguous

File: flutter/next.md [hunk 5]
```diff
@@ -622,21 +637,26 @@ Countly.disablePushNotifications();</code></pre>
   For iOS push notification please follow the instruction from this URL
   <a href="https://support.count.ly/hc/en-us/articles/360037753511-iOS-watchOS-tvOS-macOS#push-notifications">https://resources.count.ly/docs/countly-sdk-for-ios-and-os-x#section-push-notifications</a>
 </p>
+<!-- For the below file, we are told where to find it, but not what to do with it. Seems like the file is not needed in this page. Rather it is needed in the above link. While on the above link, I could not find the file. -->
 <p>
   For Flutter you can find <code>CountlyNotificationService.h/m</code> file under
   <code>Pods/Development Pods/Countly/{PROJECT_NAME}/ios/.symlinks/plugins/countly_flutter/ios/Classes/CountlyiOS/CountlyNotificationService.h/m</code><br>
   <br>
+  <!-- above line is too long. breaks the screen -->
+  <!-- Also, couldn't find this path "Pods/Development Pods" any where -->
```
Comment (turtledreams) [inline flutter/next.md:646]: ignoreable

File: flutter/next.md [hunk 6]
```diff
@@ -622,21 +637,26 @@ Countly.disablePushNotifications();</code></pre>
...
   <strong>Pro Tips to find the files from deep hierarchy:<br></strong>
 </p>
 <ul>
   <li>
     You can filter the files in the navigator using a shortcut ⌥⌘J (Option-Command-J),
     in the filter box type "CountlyNotificationService" and it will show the
     related files only.
+    <!-- Option-Command-J does nothing on my xcode -->
```
Comment (turtledreams) [inline flutter/next.md:654]: well if you say so

File: flutter/next.md [hunk 7]
```diff
@@ -622,21 +637,26 @@ Countly.disablePushNotifications();</code></pre>
...
   </li>
   <li>
     You can find the file using the shortcut ⇧⌘O (Shift-Command-O) and then navigate
     to that file using the shortcut ⇧⌘J (Shift-Command-J)<br>
+    <!-- Shift-Command-O opens a filter box but the file is not found and Shift-Command-J does not seem to do anything. -->
```
Comment (turtledreams) [inline flutter/next.md:659]: again a mac world

File: flutter/next.md [hunk 8]
```diff
@@ -622,21 +637,26 @@ Countly.disablePushNotifications();</code></pre>
   For iOS push notification please follow the instruction from this URL
   <a href="https://support.count.ly/hc/en-us/articles/360037753511-iOS-watchOS-tvOS-macOS#push-notifications">https://resources.count.ly/docs/countly-sdk-for-ios-and-os-x#section-push-notifications</a>
 </p>
+<!-- For the below file, we are told where to find it, but not what to do with it. Seems like the file is not needed in this page. Rather it is needed in the above link. While on the above link, I could not find the file. -->
```
Comment (turtledreams) [inline flutter/next.md:640]: okay seems like we only said you can find this file here and nothing else

File: android/next.md (hunk 3 above)
Comment (ArtursKadikis) [reply android/next.md:752]: we should mention that the service files are added implicitly by the SDK

**Comment by ArtursKadikis** [comment]
marked the ones to integrate on the first pass. we'll tackle the others afterwards

**Pull request closed without merging**